### ISSUE TITLE
Organiser of particular conference has the option to manage all the conferences in the dashboard

### ISSUE THREAD

**Issue by siddhantbajaj** [openSUSE/osem#1390]
Currently organiser of a particular conference has the option to manage all other conferences in the dashboard. This might be little confusing. Screenshot of this issue:
![screenshot from 2017-03-17 23 56 20](https://cloud.githubusercontent.com/assets/17237680/24057433/59438c98-0b6d-11e7-9334-f2d08e1fff70.png)

**Comment by GabrielSVinha** [comment]
On it! ;-)

**Comment by siddhantbajaj** [comment]
Great @GabrielSVinha but i will suggest to you to wait for someone's feedback. Currently no label is assigned to this issue. You should look for labelled issues.

**Comment by differentreality** [comment]
Just because the menu says manage, it doesn't mean the user can manage the other conferences (or better manage everything about the other conferences, the user can manage some parts of the conferences).

I don't understand what's the problem.

**Comment by GabrielSVinha** [comment]
@siddhantbajaj ok, i'll do it. But shouldn't this be a junior?

**Comment by siddhantbajaj** [comment]
@differentreality yes I know it just says manage but it doesn't really allow user to manage. In fact organiser of some conference can't manage anything of some other conference and rightly so. So basically my question is why are even showing user this option to manage conference where all they can do is to see the dashboard of the conference. Something like this:
![screenshot from 2017-03-20 01 32 12](https://cloud.githubusercontent.com/assets/17237680/24084281/13157df2-0d0d-11e7-84a1-ddb43a800682.png)

So the options shown are:

- Dashboard (I don't know why are we allowing organiser of conference A see how many registration, submissions conference B has)
- Basics (does not work)
- Commercial (does not work, no authorization error)
- Roles (shows type of roles and users associated with it, we can show this in some other place like in view conference)

So basically what I propose is we should not even show 'Manage B' option in menu to an organiser of conference A. :wink:

**Comment by differentreality** [comment]
We show the menu, because we want organizers to be able to view stats from other conferences. Eg. if last year conference X had most of the submissions towards the end of the cfp deadline, that gives some useful information to organizers of conference Y.

And I am kind of assuming here that the conferences might be yearly conferences of same community. This makes sense to be limited to conferences of same team/organization when https://github.com/openSUSE/mentoring/issues/72 is enabled.

That said, we should fix what isn't working.

**Comment by SidharthBansal** [comment]
I think instead of manage ,"view" should be written as other organizers ,say organizer X ,can only see to organizer Y.It can't modify the contents of organisation Y.
For the organizer X, he should see Manage X, View Y , View Z etc.

**Comment by nasia-sam** [comment]
I could work on it!
